src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
         // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        // Create model
+        Driver driver = new Driver();
+        driver.setName("Bohdan");
+        driver.setLicenseNumber("lic0285442_2456cer");
+        System.out.println(driverService.create(driver));
+        // Read data
+        System.out.println(driverService.get(1L));
+        System.out.println(driverService.getAll());
+        // Update data
+        driver.setId(1L);
+        driver.setName("Myhaylo");
+        driverService.update(driver);
+        System.out.println(driverService.getAll());
```

Comment:
avoid using 1L, because it is bed prictice. Your driver already have id. Set only name or license number

Reply:
ok